### DISCUSSION TITLE
大佬出个自定义模板，边学习边用，非自定义模式DNS是泄露的。

### DISCUSSION THREAD

**Discussion by cncot** [immortalwrt/homeproxy#158] [General]
大佬出个自定义模板，边学习边用，非自定义模式DNS是泄露的。

**Comment by xlighting2017** [comment]
交点学费，上你那个“DNS泄露”测试网站，买他的VPN，立马永久解决 “DNS泄露问题”

**Comment by muink** [comment]
模板有 搜 homeproxy-example
非自定义泄漏无解，因为并没有阻止客户端向公共dns发起请求

**Reply by rianjskis** [reply]
> 模板有 搜 homeproxy-example 非自定义泄漏无解，因为并没有阻止客户端向公共dns发起请求

请教下如果我想在自定义路由模式下.手动配置config.json文件,但是启动后依然会被覆盖,该修改哪里让启动后不被覆盖我所写的配置文件.

**Reply by muink** [reply]
无法, 手动配置config.json可以使用裸核.
裸核也可以自动配置路由和防火墙

**Reply by rianjskis** [reply]
> 无法, 手动配置config.json可以使用裸核.
> 裸核也可以自动配置路由和防火墙

好的
